cmd/search/jobs.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+)
+
+var jobBytes []byte
```

Comment:
This needs a guarding lock.

Reply:
> This needs a guarding lock.

Added as part of a8d9ce2 -> e9296ac, which also added a few more regexps, JSON `<title>`s for jobs without URIs (e.g. `ci-kubernetes-conformance-node-e2e-rhel`), explicit ignores for the `triggered` job state, and a few more early-test-phase regexps.